### DISCUSSION TITLE
Distinguish images from different folders

### DISCUSSION THREAD

**Discussion by josago97** [dotnet/maui#9666] [Q&A] [answered]
I have a MAUI project with multiple folders in the images folder. In each folder there are images with the same name. I can't find a way to reference an image within a folder.

For example, the following image shows two folders, each containing an image with the same name. Well, if I put Source="A/image.png" in the XAML in the Image tag, it doesn't show anything. And if I put Source="image.png" it shows the first one that appears in alphabetical order.
![image](https://user-images.githubusercontent.com/26380095/186808110-620da84c-fabe-4b7f-8620-d1753af8f4ae.png)

Is there a way to tell it to show the image that is in folder B?

**Comment by xwang-888** [comment, marked as answer]
Unfortunately, it is not possible at present, but you can avoid naming repetition to solve this problem.

**Reply by josago97** [reply]
Ok, thank you
